simple_salesforce/api.py

```diff
@@ -800,20 +800,27 @@ def describe_layout(self, record_id, headers=None):
             )
         return self.parse_result_to_json(result)
 
-    def get(self, record_id, headers=None):
+    def get(self, record_id, headers=None, fields=None):
         """Returns the result of a GET to `.../{object_name}/{record_id}` as a
         dict decoded from the JSON payload returned by Salesforce.
         Arguments:
         * record_id -- the Id of the SObject to get
         * headers -- a dict with additional request headers.
+        * fields -- a string with the specific fields to get
         """
+        record_url = urljoin(self.base_url, record_id)
+
+        if fields:
+            record_url += f'?fields={fields}'
+
         result = self._call_salesforce(
-            method='GET', url=urljoin(self.base_url, record_id),
+            method='GET', url=record_url,
             headers=headers
             )
         return self.parse_result_to_json(result)
 
-    def get_by_custom_id(self, custom_id_field, custom_id, headers=None):
+    def get_by_custom_id(self, custom_id_field, custom_id, headers=None,
```

Comment:
I think we can actually just refactor this to pass **kwargs in def get_by_custom_id as well as in the result = self._call_salesforce and pass **kwargs there as well. Then the user can include any potential additional parameters in the future and pass fields such that params={'fields':'field1,field2,field3,etc'}

Reply:
dug into this a bit more. I think the better option is to just pass **kwargs therefore users can just input as so

sf.Account.get('recordid',params='fields=Name,''RecordTypeId')